engine/src/main/java/com/datatorrent/stram/plan/physical/PhysicalPlan.java

```diff
@@ -979,6 +998,14 @@ public void deployChanges() {
     this.undeployOpers.removeAll(newOpers.keySet());
```

Comment:
For consistency, prefix getDependents() with "this." (though I find the unnecessary use of "this" distracting -- my mind immediately starts to scan the code for possible ambiguities when there are none).